### PR TITLE
docs: add evm summary

### PR DESCRIPTION
**Pull request by ArnaudBD**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Resolves #<Issue number>

## What is the new behavior?

-
-
-

## Other information

Time spent on this PR:

### CODE REVIEW

File: docs/evm.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+INTRO
+Ethereum Virtual Machine (EVM) lies at the heart of every Ethereum activity.
```
Comment (Eikix) [inline docs/evm.md:3]: Proposal:
>  The EVM, or Ethereum Virtual Machine, is the runtime environment for smart contracts on the Ethereum blockchain. It is responsible for executing the code of these contracts and maintaining their state. It is a virtual machine, meaning that it is a software-based simulation of a physical machine. The EVM is responsible for enforcing the rules of the Ethereum protocol and ensuring that all transactions on the network are valid.

File: docs/evm.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+You can see it as the Operating System (OS) of Ethereum. But, as you don’t need to know how an OS works to use your laptop, you don’t need to know how EVM works to use Ethereum. That is surely why you’re reading this. 
+
+Kakarot is an implementation of the EVM in Starknet.
```
Comment (Eikix) [inline docs/evm.md:6]: proposal
> Kakarot is an implementation of the EVM, currently deployed as a smart-contract on Starknet.

File: docs/evm.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+The TL;DR
+The reason Ethereum Protocol exists is to maintain the Ethereum Virtual Machine. It is hard to imagine the future of Starknet without its own implementation of EVM, this is why we are building it.
```
Comment (Eikix) [inline docs/evm.md:9]: `The reason Ethereum Protocol exists is to maintain the Ethereum Virtual Machine.`

I'm not sure this is correct.

File: docs/evm.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+Here it is what every Ethereum user should kn
```
Comment (Eikix) [inline docs/evm.md:10]: know?

File: docs/evm.md (hunk 3 above)
Comment (ArnaudBD) [reply docs/evm.md:9]: > The Ethereum protocol itself exists solely for the purpose of keeping the continuous, uninterrupted, and immutable operation of this special state machine.

source: https://ethereum.org/en/developers/docs/evm/

**Review by Eikix — APPROVED** [review]
GG!

**Pull request merged**